commands/SummarizeCommand.ts

```diff
@@ -1,149 +1,89 @@
 import {
-    IHttp,
-    IModify,
-    IRead,
-} from "@rocket.chat/apps-engine/definition/accessors";
-import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+	IHttp,
+	IModify,
+	IRead,
+} from '@rocket.chat/apps-engine/definition/accessors';
+import { IRoom } from '@rocket.chat/apps-engine/definition/rooms';
 import {
-    ISlashCommand,
-    SlashCommandContext,
-} from "@rocket.chat/apps-engine/definition/slashcommands";
-import { IUser } from "@rocket.chat/apps-engine/definition/users";
+	ISlashCommand,
+	SlashCommandContext,
+} from '@rocket.chat/apps-engine/definition/slashcommands';
+import { IUser } from '@rocket.chat/apps-engine/definition/users';
+import { notifyMessage } from '../helpers/notifyMessage';
+import { createTextCompletion } from '../helpers/createTextCompletion';
+import { createSummaryPrompt } from '../constants/CodePrompts';
+import { App } from '@rocket.chat/apps-engine/definition/App';
 
 export class SummarizeCommand implements ISlashCommand {
```

Comment:
Update class name to reflect the actual slash command, you may have named it `SummarizeCommand` while copying boiler plate of the Summarizer app.

Reply:
Have already changed the name to reflect the actual commands and functionalities as an AI programmer.